src/axom/primal/tests/primal_winding_number.cpp

```diff
@@ -476,6 +476,101 @@ TEST(primal_winding_number, degenerate_cases)
     abs_tol);
 }
 
+TEST(primal_winding_number, rational_winding_number)
```

Comment:
Minor: Now that there NURBS tests, should this one be renamed to include Bezier?
```suggestion
TEST(primal_winding_number, rational_bezier_winding_number)
```